packages/block-library/src/comment-content/block.json

```diff
@@ -35,12 +35,26 @@
 				"fontSize": true
 			}
 		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": true,
+				"color": true,
+				"width": true,
+				"style": true
+			}
+		},
 		"spacing": {
 			"padding": [ "horizontal", "vertical" ],
 			"__experimentalDefaultControls": {
 				"padding": true
 			}
 		},
 		"html": false
-	}
+	},
+	"editorStyle": "wp-block-comment-content-editor",
```

Comment:
With the removal of the editor stylesheet, should this also be removed?

Reply:
Removed editorStyle in json file.